sympy/concrete/summations.py

```diff
@@ -254,6 +254,28 @@ def _eval_derivative(self, x):
         else:
             return NotImplementedError('Lower and upper bound expected.')
 
+    def _eval_integral(self, x):
+        if isinstance(x, Symbol) and x not in self.free_symbols:
+            return x*self
+
+        f, limits = self.function, list(self.limits)
+
+        limit = limits.pop(-1)
+
+        if limits:
```

Comment:
I mean in the test suite. Both of the tests you added have a single summation variable.